webviz_subsurface/_datainput/pvt_data.py

```diff
@@ -0,0 +1,112 @@
+########################################
+#
+#  Copyright (C) 2020-     Equinor ASA
+#
+########################################
+
+import pandas as pd
+from opm.io.ecl import EclFile
+
+from webviz_config.common_cache import CACHE
+from webviz_config.webviz_store import webvizstore
+
+from .opm_init_io import Oil, Gas, Water, DryGas
+
+try:
+    from .fmu_input import load_ensemble_set
+except ImportError:
+    from fmu_input import load_ensemble_set
```

Comment:
Is the need for this `try`/`except` due to `__main__` below?

Reply:
Yes, I agree that a consistent style makes sense. When removing the `__main__` if case, this `try/except` statement can be removed as well.